mobify/platform-scaffold
# Update logic for when to show the saved address shipping form

This PR fixes a bug where newly registered users would not see the shipping address form on the shipping page of the checkout. *Note:* Saved addresses for the SFCC connector have not been implemented yet, they will be fixed in another sprint (https://mobify.atlassian.net/browse/WEBDATA-74)

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-48
 **Linked PRs**: n/a

## Changes
- Check length of savedAddresses prop before rendering saved address options

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- register a new user
- add an item to the cart and proceed to the shipping page
- you should see the full shipping address form (not a radio button that says "Add New Shipping Address"
- switch to the SFCC connector (in main.jsx uncomment the sfcc connector and comment out the merlins one)
- Preview the [SFCC Site](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- register a new user
- add an item to the cart and proceed to the shipping page
- you should see the full shipping address form (not a radio button that says "Add New Shipping Address"
